packages/govuk-frontend/src/govuk/helpers/_typography.scss

```
@@ -62,6 +62,22 @@
   font-weight: $govuk-font-weight-bold if($important, !important, null);
 }
 
+/// Tabular number helper
+///
+/// Using font-feature-settings is generally discouraged, but unfortunately
+/// IE 11 doesn't support font-variant-numeric or @supports, so we resort to
+/// using font-feature-settings as the default and then removing it again if
+/// font-variant-numeric is supported.
+///
+/// @param {Boolean} $important [false] - Whether to mark declarations as
+///   `!important`. Generally Used to create override classes.
+/// @access public
+
+@mixin govuk-typography-tabular-numbers($important: false) {
+  font-feature-settings: "tnum" 1 if($important, !important, null);
+  font-variant-numeric: tabular-nums if($important, !important, null);
```

Was there a risk at the time that `font-feature-settings` would be dropped?

Only checked Safari and Firefox but `font-feature-settings` without `font-variant-numeric` still works

```suggestion
```

---

Or drop `font-feature-settings` and use ~`font-variant-numeric`~ `font-variant` for IE11 support?

```suggestion
  font-variant: tabular-nums if($important, !important, null);
```

---

`font-feature-settings` isn't at risk of being dropped, but using it is discouraged these days due to its broadness and tendency to overreach, as I described in the thoughts section, as well as the sometimes obtuse syntax. 

Newer, more bespoke properties like `font-variant-numeric` were introduced to solve those problems, but IE doesn't support them as a result.

CanIUse doesn't seem to mention this, but IE 11 only supports the `small-caps` keyword for `font-variant`. `font-variant: tabular-nums` unfortunately doesn't do anything there.

---

Ah that's a shame it's only `small-caps`, was hoping we could have the best of both worlds

I'd be happy dropping IE11 support since we'd still default to proportional not monospaced numbers

Character count text updates would jump a bit in IE11 (variable number width) but I think that's acceptable